### ISSUE TITLE
[0.9.703] Bots - Does not account for docked Shuttles being in the way

### ISSUE THREAD

**Issue by Kaillera** [Regalis11/Barotrauma#2173]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
I have a bot who is trying to find a quicker route around the ship from the right wing of my sub. They are trying to get around by swimming from an airlock to my Shuttle's docking hatch, which is blocked by my shuttle. Although he can get into the shuttle from the bottom and climb up from it, he decides to wedge himself inbetween the shuttle and sub.

![](https://user-images.githubusercontent.com/5086682/73950288-ac073000-48c9-11ea-8c91-f540e6f81733.PNG)

Although even if he did climb into the shuttle, this will lead to another issue #2174.

**Version**
0.9.703

**Additional Information**
I don't think the bot should be trying to use water waypoints in the first place unless they were forced out of it.

**Comment by itchyOwl** [comment]
Fixed in Regalis11/Barotrauma-development@680935a and by adding more waypoints around the docking area.

**Comment by Rokvach** [comment]
No issues, closing.
